spec.bs

```diff
@@ -548,6 +553,9 @@ input {{SecurePaymentConfirmationRequest}} |data|, are:
 1. If |data|["{{SecurePaymentConfirmationRequest/credentialIds}}"] is empty,
     return `false`.
 
+1. If |data|["{{SecurePaymentConfirmationRequest/rpId}}"] is not a
+    [valid domain](https://url.spec.whatwg.org/#valid-domain), return `false`.
```

Comment:
Normally I would say to do this via an [anchor block](https://tabatkins.github.io/bikeshed/#custom-dfns) (e.g. see our existing block on line 24), but interestingly enough it looks like WebAuthn does it via a [link-default](https://tabatkins.github.io/bikeshed/#link-defaults) block instead...

So let's [copy WebAuthn](https://github.com/w3c/webauthn/blob/main/index.bs#L313) for now, and we can ask Dom to take a look at the PR when ready.

Reply:
Cool! Done.